.github/workflows/cover.yaml

```
@@ -28,5 +28,5 @@ jobs:
         with: |
           file: ./coverage.out
           fail_ci_if_error: true
-        env:
-          CODECOV_TOKEN: ${{ secrets.CODECOV_TOKEN }}
+          env:
+            CODECOV_TOKEN: ${{ secrets.CODECOV_TOKEN }}
```

```suggestion
          token: ${{ secrets.CODECOV_TOKEN }}
```
see https://github.com/codecov/codecov-action/blob/main/README.md

---

Or https://github.com/codecov/codecov-action/tree/v3.1.6 for v3